cloud_info_provider/core.py

```diff
@@ -11,10 +12,11 @@
     'openstack': 'cloud_info_provider.providers.openstack.OpenStackProvider',
     'opennebula': 'cloud_info_provider.providers.opennebula.'
                   'OpenNebulaProvider',
-    'indigoon': 'cloud_info_provider.providers.opennebula.IndigoONProvider',
+    'indigoon': 'cloud_info_provider.providers.indigoon.IndigoONProvider',
```

Comment:
If the `IndigoONProvider` is now quite similar to the main `OpenNebulaProvider` it may be appropriate to drop it as it was created for legacy reasons.

Reply:
@alvarolopez is there any usage of IndigoONProvider in DEEP? If not I think it's safe to drop it, if yes, we could check if it's bringing any added value (to start with I doubt it does what it is supposed to do, see https://github.com/EGI-Foundation/cloud-info-provider/pull/112/commits/d748e683bb5f71e78cf44a0bbe662b55b6b0881c#diff-b1f283932733d6721cb621901c205e01R19)

Could you comment on this?

Reply:
@sustr4 could you also comment?